client/src/store/index.ts

```diff
@@ -0,0 +1,18 @@
+import create, { GetState } from 'zustand'
+import { devtools, NamedSet } from 'zustand/middleware';
+/**
+// Allows for cross-slice
```

Comment:
Any reason why these are commented out?

Reply:
Boilerplate, theres nothing we are using state management for right now so i kept this boilerplate because we'll need it to combine the different store slices later.

The full example is on the `f/zustand-example` branch

Reply:
I recommend keeping boilerplate in documentation or a separate snippet directory instead, not a big fan of having unused snippets in the core code

Reply:
I'm gonna uncomment most of this and make it ready to be used (just needs to have slices added and `useStore` called). 

Ill let the rest of the team give some input for this since I think its not a big deal to keep it here, we will be using this soon anyways. 

@NguyenPatrick @andrewpratheepan @Wuon